### PR TITLE
User can see hand rankings

### PR DESCRIPTION
**Pull request by ericliu121187**
Release Notes

User can see hand rankings now. Also, there are other updates in this one, I'll point them out in review.

Screenshots

https://share.getcloudapp.com/4guyQqeK

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@
     <dependency>
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
-      <version>1.16.20</version>
+      <version>1.18.12</version>
```
Comment (ericliu121187) [inline backend/pom.xml:62]: Bumping up lombok

File: frontend/.eslintrc [hunk 2]
```diff
@@ -73,6 +73,7 @@
   "settings": {
     "import/resolver":  {
       "node": {
+        "extensions": [".js", ".ts", ".tsx"],
```
Comment (ericliu121187) [inline frontend/.eslintrc:76]: I integrated Typescript, so we can use that moving forward. This is for eslint.

File: frontend/.flowconfig [hunk 3]
```diff
@@ -8,6 +8,9 @@ src/flow/types.js
 [lints]
 
 [options]
+module.file_ext=.js
+module.file_ext=.ts
+module.file_ext=.tsx
```
Comment (ericliu121187) [inline frontend/.flowconfig:13]: Integrating TS with Flow - we want to slowly deprecate TS

File: frontend/package.json [hunk 4]
```diff
@@ -2,36 +2,47 @@
...
+    "start": "yarn build:style && react-scripts start",
+    "build": "yarn build:style && react-scripts build",
     "lint": "eslint src --fix",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "deploy": "yarn build && aws s3 sync build/ s3://short-deck-poker"
+    "deploy": "yarn build && aws s3 sync build/ s3://short-deck-poker",
+    "clean-install": "rm -rf node_modules && yarn",
+    "build:style": "postcss src/styles/tailwind-imports.css -o src/styles/tailwind.css",
+    "watch:style": "postcss src/styles/tailwind-imports.css -o src/styles/tailwind.css -w"
```
Comment (ericliu121187) [inline frontend/package.json:37]: We are adding [TailwindCSS](https://tailwindcss.com/), moving forward. It's easy and modern.

**Comment by teddylo57** [comment]
Looks good

**Pull request merged**